scm/github/github.go

```diff
@@ -53,7 +53,7 @@ type client struct {
 	AuthReq *github.AuthorizationRequest
 }
 
-// New returns a Source implementation that integrates with
+// New returns a SCM implementation that integrates with
```

Comment:
super nitpicky, but this is the first instance i found where "SCM" is actually all caps so it jumped out at me for being inconsistent 🤷🏼‍♂️

Reply:
Umm... it should be in all instances `S` was capitalized for source.

Reply:
For example here it is in another file:
https://github.com/go-vela/server/pull/536/files#diff-20b08cad4aeb414832e6f834e33e07c46c1be25d5b3fc3d55d602a8fa0d925a5L23


Can definitely modify it so there is no all caps tho, if that is preferred.

Reply:
like i said, nitpicky.. i approved the PR so do what you will 🤪

Reply:
Gramatically, I prefer all caps here, but if we're going that far it would be:
```suggestion
// New returns an SCM implementation that integrates with
```
:zany_face:

Reply:
+1 for @cognifloyd